src/System.Private.CoreLib/shared/System/Globalization/NumberFormatInfo.cs

```diff
@@ -384,13 +385,15 @@ public static NumberFormatInfo CurrentInfo
                 System.Globalization.CultureInfo culture = CultureInfo.CurrentCulture;
                 if (!culture._isInherited)
                 {
-                    NumberFormatInfo info = culture._numInfo;
+                    NumberFormatInfo? info = culture._numInfo;
                     if (info != null)
                     {
                         return info;
                     }
                 }
-                return ((NumberFormatInfo)culture.GetFormat(typeof(NumberFormatInfo)));
+                // returns non-nullable when passed typeof(NumberFormatInfo)
+                NumberFormatInfo? nfi = (NumberFormatInfo?)culture.GetFormat(typeof(NumberFormatInfo));
```

Comment:
can't we do: `return (culture.GetFormat(typeof(NumberFormatInfo)) as NumberFormatInfo)!`

Reply:
technically yes but there is too many special characters and brackets on that line so I prefer to keep it separate

Reply:
actually, no need for extra brackets so changing